soweego/commons/constants.py

```diff
@@ -164,15 +162,22 @@
 WIKIDATA_API_SESSION = 'wiki_api_session.pkl'
 SHARED_FOLDER = '/app/shared/'
 
+
 # Supervised classification
+SVC_CLASSIFIER = 'support_vector_machines'
```

Comment:
```suggestion
SVC_CLASSIFIER = 'support_vector_machines'
LINEAR_SVC_CLASSIFIER = 'linear_support_vector_machines'
```
I would recommend not to drop support for `LinearSVC`.